tests/Dfe.PlanTech.Web.E2ETests/cypress/commands/login.js

```diff
@@ -8,11 +8,11 @@ Cypress.Commands.add("login", ({ email, password, url }) => {
         return false
     });
     
-    cy.visit(url);
-    cy.get("input#username").type(email);
-    cy.get("input#password").type(password);
-    cy.get("div.govuk-button-group button.govuk-button").first().click();
-    cy.url().should("contain", url);
+  cy.visit(url);
+  cy.get("input#username").type(email);
+  cy.get("input#password").type(password);
+  cy.get("div.govuk-button-group button.govuk-button").first().click();
+  cy.wait(4000);
```

Comment:
why the added wait? - were you having problems when testing locally?

Reply:
Yes, it was checking the URL of the self-assessment page before the page load so put it in to counter that but I've taken that URL check out (there are still url checks within each page's test suite so nothing lost from removing it from the login function). Have taken the wait out now.

Reply:
Put wait back in due to failing checks - will investigate on a new branch